src/components/admin/AppLayout.vue

```diff
@@ -41,7 +43,7 @@
         'isLoading'
       ]),
       breadcrumbs () {
-        return this.$store.getters.breadcrumbs(this.$route.name)
+        return utils.findInNestedByName(Breadcrumbs, this.$route.name)
```

Comment:
move findNestedByName into breadcrumbs component. pass configuration object and current route as properties